### ISSUE TITLE
Hyperparametric problems

### ISSUE THREAD

**Issue by chengtao-lv** [yue-zhongqi/ifsl#10]
under your instructions, i run **python main.py --dataset miniImagenet --method metatrain --train_aug --test maml5_resnet** 
and finally get the result of 66.5%,far from the result in your paper  about 70%, is the Hyperparametric problems?
ps: I did not use the novel.hdf5 file,because I don't think this file is getting anything useful,does file make the situation?

**Comment by chengtao-lv** [comment]
And I must add that the inner and outer learning rate in your paper about maml are both 0.01,but in your code are 0.001 and 0.005.

**Comment by yue-zhongqi** [comment]
Hi, which pre-trained backbone did you use?

**Comment by chengtao-lv** [comment]
![image](https://user-images.githubusercontent.com/60501594/140467965-69dbf68f-a17b-47c0-a74a-40d7274da5eb.png)

hey！ resnet10，do I download or use the wrong backbone?

**Comment by chengtao-lv** [comment]
thanks for your reply, appreciate it!

**Comment by yue-zhongqi** [comment]
I used the Resnet10 pre-trained using Simpleshot. Please download here:
https://drive.google.com/file/d/1wWFQibiLL6yJWNXsHIxioucp0DnxOelP/view?usp=sharing

**Comment by chengtao-lv** [comment]
是啊，哥们。我就是从这里下载的，导入也没有啥问题，就是达不到你的效果.....方便加一个wx吗？如果不介意的话可以将wx号发我邮箱，897674362@qq.com。

**Comment by nancy-ux** [comment]
> under your instructions, i run **python main.py --dataset miniImagenet --method metatrain --train_aug --test maml5_resnet** and finally get the result of 66.5%,far from the result in your paper about 70%, is the Hyperparametric problems? ps: I did not use the novel.hdf5 file,because I don't think this file is getting anything useful,does file make the situation?

hello,I want to reproduce results 76.37% from the paper (MAML + ifsl with dataset miniImagenet and model ResNet10 ). But when I run an command (python main.py --dataset miniImagenet --model na --method metatrain --train_aug --test maml5_ifsl_resnet ),get a wrong answer whose acc=32. 
Did you reproduce the result successfully? Could you tell me how to get the 76.37% from the paper?If my command was wrong , what is the correct command? thank you!
